src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -217,6 +227,9 @@ export class ContentTypeFieldsDropZoneComponent extends BaseComponent implements
 
     private toggleDialog(): void {
         this.displayDialog = !this.displayDialog;
+        setTimeout(() => {
+            this.dialogActiveTab = 0;
```

Comment:
Why the timeout?

Reply:
I'm not following how it can be done in the after view init, since the component it's already loaded

Reply:
https://angular.io/api/core/AfterViewInit